state/indexer/block/kv/export_testing.go

```diff
@@ -0,0 +1,5 @@
+package kv
+
+func GetKeys(indexer BlockerIndexer) [][]byte {
```

Comment:
Is this a function only used for unit tests? If that is the case, it is currently also being included when building the `cometbft` binary. If you want it to only be present when building UT binaries (i.e., when running `go test`) so should, either rename this file to have suffix "_test.go", or move this code to one of such files.

Reply:
Ahh, this is a tough one. It is used outside `kv` and `kv_test` packages, namely in `package state_test` in [state/pruner_test.go](https://github.com/cometbft/cometbft/blob/5b2522ff35ac86a28c703071c07bf72f9bd49124/state/pruner_test.go).

So renaming the file to "*_test.go" would make it invisible there, and it's not possible to move it outside `kv` package since it accesses private function `getKeys`.

Reply:
I see. Quite unfortunate, but I guess I can't think of any option than what you've done.